--- origsrc/libindicate-0.3.6/gtk-doc.local.make	2010-02-04 17:22:10.000000000 -0600
+++ src/libindicate-0.3.6/gtk-doc.local.make	2010-05-03 01:01:03.235113400 -0500
@@ -117,7 +117,7 @@ html-build.stamp: sgml.stamp $(DOC_MAIN_
 	cd $(srcdir)/html && gtkdoc-mkhtml $(mkhtml_options) $(MKHTML_OPTIONS) $(DOC_MODULE) ../$(DOC_MAIN_SGML_FILE)
 	test "x$(HTML_IMAGES)" = "x" || ( cd $(srcdir) && cp $(HTML_IMAGES) html )
 	@echo 'gtk-doc: Fixing cross-references'
-	cd $(srcdir) && gtkdoc-fixxref --module-dir=html --html-dir=$(HTML_DIR) $(FIXXREF_OPTIONS)
+	cd $(srcdir) && gtkdoc-fixxref --module=$(DOC_MODULE) --module-dir=html --html-dir=$(HTML_DIR) $(FIXXREF_OPTIONS)
 	touch html-build.stamp
 
 ##############
--- origsrc/libindicate-0.3.6/libindicate/Makefile.am	2010-03-11 16:09:20.000000000 -0600
+++ src/libindicate-0.3.6/libindicate/Makefile.am	2010-05-03 00:48:06.192669100 -0500
@@ -111,7 +111,7 @@ BUILT_GIRSOURCES =
 
 irscanner_headers = $(patsubst %,$(srcdir)/%,$(indicate_headers))
 
-Indicate-0.2.gir: $(INTROSPECTION_SCANNER) $(irscanner_headers) Makefile.am
+Indicate-0.2.gir: $(INTROSPECTION_SCANNER) $(irscanner_headers) libindicate.la Makefile.am
 	$(INTROSPECTION_SCANNER) \
 		-v --namespace Indicate \
 		--nsversion=0.2 \
@@ -119,7 +119,7 @@ Indicate-0.2.gir: $(INTROSPECTION_SCANNE
 		--include=GObject-2.0 \
 		--include=GLib-2.0 \
 		--include=DbusmenuGlib-0.2 \
-		--library=indicate \
+		--library=libindicate.la \
 		$(LIBINDICATE_CFLAGS) -I $(srcdir) -I $(top_srcdir) \
 		--output Indicate-0.2.gir $(irscanner_headers)
 
--- origsrc/libindicate-0.3.6/libindicate-gtk/Makefile.am	2010-03-11 16:09:20.000000000 -0600
+++ src/libindicate-0.3.6/libindicate-gtk/Makefile.am	2010-05-03 00:47:58.432225200 -0500
@@ -47,7 +47,7 @@ BUILT_GIRSOURCES =
 
 irscanner_headers = $(patsubst %,$(srcdir)/%,$(indicate_gtk_headers))
 
-Indicate-Gtk-0.2.gir: $(INTROSPECTION_SCANNER) $(irscanner_headers) Makefile.am
+Indicate-Gtk-0.2.gir: $(INTROSPECTION_SCANNER) $(irscanner_headers) libindicate-gtk.la Makefile.am
 	$(INTROSPECTION_SCANNER) \
 		-v --namespace Indicate \
 		--nsversion=0.2 \
@@ -56,7 +56,7 @@ Indicate-Gtk-0.2.gir: $(INTROSPECTION_SC
 		--include=GLib-2.0 \
 		--include=GdkPixbuf-2.0 \
 		--include=Indicate-0.2 \
-		--library=indicate-gtk \
+		--library=libindicate-gtk.la \
 		--add-include-path=$(top_builddir)/libindicate \
 		$(LIBINDICATEGTK_CFLAGS) -I$(srcdir) -I$(top_srcdir) \
 		--output Indicate-Gtk-0.2.gir $(irscanner_headers)
